CHANGELOG.md

```diff
@@ -37,7 +37,25 @@ Bottom level categories:
 - Hal
 -->
 
-## Unreleased
+## 22.0.0 (2024-07-17)
+
+### Overview
+
+### Our first major version release!
+
+For the first time ever, WGPU is being released with a major version (i.e., 22.* instead of 0.22.*)! Maintainership has decided to fully adhere to [Semantic Versioning](https://semver.org)'s recommendations for versioning production software. According to [SemVer 2.0.0's own Q&A about when to use 1.0.0 and beyond](https://semver.org/spec/v2.0.0.html#how-do-i-know-when-to-release-100):
+
+> ### How do I know when to release 1.0.0?
+> 
+> If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backward compatibility, you should probably already be 1.0.0.
+
+It is a well-known fact that WGPU has been used for applications and platforms already in production for years, at this point. We are often concerned with tracking breaking changes, and affecting these consumers' ability to ship. By releasing our first major version, we publicly acknowledge that this is the case. We encourage other projects in the Rust ecosystem to follow suit.
+
+### Overview
+
+A major ([pun intended](#our-first-major-version-release)) theme of this release is incremental improvement. Beyond the typically large set of bug fixes, new features, and other adjustments to WGPU, @wumpf and @teoxoy have merged many simplifications to WGPU's internals. Many of these change WGPU to use atomically reference-counted resource tracking (i.e., `Arc<…>`), rather than using IDs to manage the lifetimes of platform-specific graphics resources of separate reference counts. This has led us to diagnose and fix many long-standing bugs, and net some neat performance improvements on the order of 30% of some worklaods.
```

Comment:
I _am_ flattered 😊, but also worth pointing out that we also got lots of fixes from other contributors! :)
(the call for help below has a bit of that vibe, but maybe make it a "join all these other contributors" in doing so?)

Reply:
So maybe something like this?

```suggestion
A major ([pun intended](#our-first-major-version-release)) theme of this release is incremental improvement. Among the typically large set of bug fixes, new features, and other adjustments to WGPU by the many contributors listed below, @wumpf and @teoxoy have merged a series of many simplifications to WGPU's internals and, in one case, to the render pass recording API. Many of these change WGPU to use atomically reference-counted resource tracking (i.e., `Arc<…>`), rather than using IDs to manage the lifetimes of platform-specific graphics resources in a registry of separate reference counts. This has led us to diagnose and fix many long-standing bugs, and net some neat performance improvements on the order of 30% of some workloads.
```

Reply:
Sounds good

Reply:
- [ ] suggestion: Let's get a link to the claim of perf. improvements. 😀

Reply:
👍

Reply:
I see @teoxoy has added an entry in the `CHANGELOG` in a PR landed after this was started. I'll rebase, and refer to that entry here instead of linking.